pkgs/data/fonts/google-fonts/default.nix

```diff
@@ -18,10 +20,6 @@ stdenv.mkDerivation rec {
     chmod -x $dest/*.ttf
   '';
 
-  outputHashAlgo = "sha256";
-  outputHashMode = "recursive";
-  outputHash = "0q03gg0sh2mljlbmhamnxz28d13znh9dzca84p554s7pwg6z4wca";
```

Comment:
I think the `fetchFromGitHub` should fix the build failures as Github regenerates archives with different timestamps.